README.md

```diff
@@ -23,45 +34,60 @@ To run Unvanquished Tray in the foreground, execute:
 unvtray
 ```
 
-or, if the above does not work (e.g. when `~/.local/bin` is not in your `PATH`):
+or, if the above does not work (e.g. when `~/.local/bin` is not in your `PATH`
+on Unix-like platforms):
 
 ```
 python -m unvtray
 ```
 
-To start it with your window manager, append ` &` as necessary (for running in
-the background) and add the command to your startup file (e.g. `.xinitrc`).
+### Autostart
 
-## Troubleshooting
+#### on GNU/Linux or BSD
```

Comment:
For now Unvanquished doesn't run on BSD (unless there is some Linux compatibility mode? I never tried that). So even if that tray browser would, this may be confusing.